src/Symfony/Component/HttpKernel/Tests/KernelTest.php

```diff
@@ -522,7 +522,7 @@ public function getNamespace(): string
             }
 
             /**
-             * @return false
+             * @return string|false
```

Comment:
Depending on the outcome of my previous comment, this might need to become `@return string|bool`?